src/main/java/at/hannibal2/skyhanni/config/commands/Commands.kt

```diff
@@ -325,6 +327,35 @@ object Commands {
             "shignore",
             "Add/Remove a user from your"
         ) { PartyChatCommands.blacklist(it) }
+
+        registerCommand("warp", "") { args: Array<String> ->
```

Comment:
put the command functions into TransferCooldown.kt as to not clutter the Commands object.